### DISCUSSION TITLE
Join Table di R

### DISCUSSION THREAD

**Discussion by DhiaFS** [teamalgoritma/community#65] [Q&A] [answered]
Saya mau tanya bagaimana cara join table di R

**Comment by inytss** [comment, marked as answer]
Dear @DhiaFS ,
Untuk join table Ibu dapat menggunakan fungsi `left_join()` yang disediakan oleh package `dplyr`.
Untuk keterangan lebih lanjut, Ibu dapat melihat diskusi pada tiket berikut https://github.com/teamalgoritma/community/discussions/59
Semoga membantu :)
